src/nccl_ofi_net.c

```diff
@@ -856,8 +847,6 @@ static ncclResult_t ofi_listen(int dev, void *handle, void **listenComm)
 
 	*listenComm = lComm;
 
-	goto exit;
```

Comment:
Again, this is required to ensure that we do not unnecessarily release already released `nccl_ofi_lock` and `lComm`.